### ISSUE TITLE
Move Old Wiki Docs To MonoGame Docs

### ISSUE THREAD

**Issue by tomspilman** [MonoGame/MonoGame#4080]
We need one or more people to go thru the old wiki docs and decide what content there is worth moving over to the new MonoGame docs in the repo:

Old Wiki Docs: https://github.com/mono/MonoGame/wiki
New Docs: https://github.com/mono/MonoGame/tree/develop/Documentation

It should be done only a page or two at a time.  Please chime in here first if you plan to work on particular pages from the wiki.

Once the doc PR is merged we can delete those old pages from the wiki.  Eventually we should end up with an empty wiki simply pointing people to the official docs on the website.

**Comment by emersont1** [comment]
Hi
I'm in the process of migrating the git pages all into on page, but there seems to be conflicting information in [[GIT'n Some Updates]] and [[GIT'ing Started with git]]

**Comment by tomspilman** [comment]
That probably needs to be re-written anyway.  I say pick whichever you think is more correct and we can revisit it again later.

**Comment by emersont1** [comment]
Okay. Just going to commit in a bit


Sent by Outlook for Android





On Thu, Nov 5, 2015 at 1:22 AM -0800, "Tom Spilman" <notifications@github.com> wrote:
That probably needs to be re-written anyway.  I say pick whichever you think is more correct and we can revisit it again later.

---

**Comment by ralfjansson** [comment]
I will try to help out.

**Comment by ralfjansson** [comment]
I have perused the old wiki and compared it with the new documentation:

1. The tutorial pages may be usefull to salvage for the "Platform-Specifics Notes" pages of
the new documentation. So do too the pages on "PCL project support", "Performance tips", 
"Properly supporting android ARM devices", "Android manifest requirements" and "Migrating to OpenTK 1.0".
2. I find the page on "Released games" interesting, so how about including that?

The Wiki gages on Git, Coding guidelines, content pipeline, contributing, effects and shaders is perhaps of less interest to include in the new documentation.

I would be happy to make an effort rewriting 1 and 2 as soon as I am able.

**Comment by tomspilman** [comment]
First thanks for any help you can give in docs.

> 1.The tutorial pages may be usefull to salvage for 

We just have to be certain that info is still accurate.  If you can at least get the PR submitted with them others can come in and read to be sure nothing is missed.

>  2.I find the page on "Released games" interesting, so how about including that?

I think we should leave marketing for the main website and not the docs.  We already represent some of the biggest and most well known titles on the home page.  A lot of those links in the wiki are dead anyway or to old titles released over 5 years ago now.

**Comment by ralfjansson** [comment]
Ok.  I will take a look at task 1. above.
Sorry for asking, but just to be sure. With "PR" you mean Pre-Requisites that can be found here and there in the old wiki?

**Comment by cra0zy** [comment]
PR means "pull request".

**Comment by ralfjansson** [comment]
Thank you!

**Comment by ralfjansson** [comment]
I have made my first PR. 
tried copy text from wiki info documents. #4507
Please let me know if I got it right.

**Comment by ralfjansson** [comment]
New PR made. More android information #4541.

**Comment by ralfjansson** [comment]
I am using the GitHub Desktop tool.

Is it possible to perhaps speed up the process by branching several work branches from the development branch? And then try make a PR each one of them?

**Comment by cra0zy** [comment]
@ralfjansson I highly recommend you listen to the following 2 tutorials:
 - Part 1 https://www.youtube.com/watch?v=0fKg7e37bQE
 - Part 2 https://www.youtube.com/watch?v=oFYyTZwMyAg

**Comment by ralfjansson** [comment]
Thank you! I will.

**Comment by ralfjansson** [comment]
What do we want to do with the coding guide lines?
https://github.com/mono/MonoGame/wiki/Coding-Guidelines

**Comment by ralfjansson** [comment]
I added a reference to the new doc here
https://github.com/mono/MonoGame/wiki/Content-Build-Pipeline
And removed the text of that page. It looked obsolete and contained some plans for developement. I hop that was okey.

**Comment by ralfjansson** [comment]
I added a reference to  the new docs for this page too:
https://github.com/mono/MonoGame/wiki/Contributing-to-MonoGame
I don't Think I am quilified to change or copy this text yet, so I have left it as it is.

**Comment by ralfjansson** [comment]
The same goes for this page:
https://github.com/mono/MonoGame/wiki/Contribution-Guidelines

Should there be page  for contributing in the new docs? Could not find such page...

**Comment by ralfjansson** [comment]
New PR made. #4607. Copied the FAQ text from the wiki to the new docs.

**Comment by tomspilman** [comment]
> What do we want to do with the coding guide lines?
> https://github.com/mono/MonoGame/wiki/Coding-Guidelines

@KonajuGames - Can you review that and make sure it all still applies?

If so we should move them into like a `CODEGUIDE.md` in the root of the repo like the `CONTRIBUTING.md` or `README.md`.  I just want to wait before we add content for contributors into the docs.

**Comment by tomspilman** [comment]
> And removed the text of that page. It looked obsolete and contained some plans for developement. I hop that was okey.

Yeah... that seems fine.

> https://github.com/mono/MonoGame/wiki/Contributing-to-MonoGame
> https://github.com/mono/MonoGame/wiki/Contribution-Guidelines

Those have been replaced by...

https://github.com/mono/MonoGame/blob/develop/CONTRIBUTING.md

... you can clear them out and point them there.  Same as with the coding guidelines lets not link that into the documentation just yet.

**Comment by ralfjansson** [comment]
New PR made #4707. "Getting Effect .fx files to compile and run Hints, Tips and Gotchas ".
Cleared up the text a bit in case you want to reuse it.
Cleared the orignal wiki page and added reference to the new doc.

**Comment by ralfjansson** [comment]
Cleared the wiki pages below:
https://github.com/mono/MonoGame/wiki/Contributing-to-MonoGame
https://github.com/mono/MonoGame/wiki/Contribution-Guidelines

**Comment by ralfjansson** [comment]
I noticed that some texts 'jump' in level for headers like in

'* Level 1 header
*** Level 2 header'

Instead of 

'* Level1 header
** Level 2 header'

Was this intentional by the author? Or should change these?

**Comment by ralfjansson** [comment]
So with regard to  #4707, is it still okey for me to continue suggesting PRs for the rest of the old wiki docs, or should we try a different strategy?

**Comment by tomspilman** [comment]
FYI.  Because of confusion we've disabled the old wiki entirely.

You should be able to access the old docs from the wiki repo history:

https://github.com/MonoGame/MonoGame.wiki.git

**Comment by ralfjansson** [comment]
Ok. Tanks :)
